.github/workflows/backend_pull_requests.yaml

```
@@ -99,18 +123,24 @@ jobs:
       - name: Run Conditional Playground Build via Make
         env:
           # Required variables for playground builds
-          AXELIX_SBS_AUTH_JWT_ALGORITHM: HMAC512
-          AXELIX_SBS_AUTH_JWT_SIGNING_KEY: ${{ env.SANDBOX_JWT_SIGNING_KEY }}
+#          AXELIX_SBS_AUTH_JWT_ALGORITHM: HMAC512
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)